tests/rules/tainted_anon_type.yaml

```diff
@@ -0,0 +1,11 @@
+rules:
```

Comment:
`tests/rules/taint_...` instead of `tainted_....`, there are a few tests that break the convention already, no big deal but FYI.

Reply:
renamed.